Packs/PrismaSaasSecurity/Integrations/SaasSecurity/SaasSecurity.yml

```
@@ -137,7 +137,7 @@ script:
     description: Retrieves incidents from the SaaS Security platform.
     arguments:
     - name: limit
-      description: The number of incidents to pull. Maximum is 200, minimum is 10. Default is 50.
+      description: "The number of incidents to pull. Important: The limit value can range from 10 to 200 and must be in multiples of 10. Default is 50."
```

```suggestion
      description: "The number of incidents to pull. Important: The limit value can range from 10 to 200 and must be in multiples of 10."
```
Since the default value is listed below, when you generate the readme, it will automatically be included in the description.